src/pages/Login.vue

```diff
@@ -214,12 +214,17 @@ export default {
         this.requestedOtp = false;
       }
     },
+    requestedOtp()
+    {
+       if (!this.resendOTPTimer) clearInterval(this.interval);
+    },
   },
   data() {
     return {
       phoneInput: "", // phone input text
       otpInput: "", // otp input text
       resendOTPTimer: 0, // the count of the timer
+      interval: 0, // to reset the timer

```

Comment:
clearer name and description - please spend significant time thinking about variable names, function names and comments - whether they would be clear to someone who is reading your code for the first time!

```
otpTimerInterval: 0, // to reset the otp timer
```